pkgs/development/python-modules/loguru/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, isPy2, pytest, colorama }:
+
+buildPythonPackage rec {
+  pname = "loguru";
+  version = "0.3.0";
+disabled = isPy2
```

Comment:
forgot a semicolon :(, my bad.
```suggestion
  disabled = isPy2;
```

Reply:
Sorry, I should have looked more carefully.